NOTICE

```diff
@@ -73,4 +73,9 @@ Copyright (2023) Databricks, Inc.
 https://github.com/databrickslabs/blueprint/blob/main/LICENSE
 
 Databricks Connect - https://pypi.org/project/databricks-connect/
-Copyright (2019) Databricks Inc.
\ No newline at end of file
+Copyright (2019) Databricks Inc.
+
+cryptography - https://github.com/pyca/cryptography
```

Comment:
move this to a proper section - under Apache 2.0. this section is for Databricks License

Reply:
Sorry may bad. I though everything below is Apache 2.0
Done.